src/test/java/org/gridsuite/study/server/NetworkModificationTreeTest.java

```diff
@@ -737,4 +761,27 @@ private String getNodeName(UUID studyUuid) throws Exception {
                 .getResponse()
                 .getContentAsString();
     }
+
+    @Test
+    public void testGetNetworkModificationsNode() throws Exception {
+        RootNode root = createRoot();
+        NetworkModificationNode node = buildNetworkModification("modification node 1", "", UUID.randomUUID(), VARIANT_ID, LoadFlowStatus.NOT_DONE, null, UUID.randomUUID(), BuildStatus.BUILT);
+        createNode(root.getStudyId(), root, node);
+
+        String bodyError = mockMvc.perform(get("/v1/studies/{studyUuid}/nodes/{nodeUuid}/network-modification/modifications", root.getStudyId(), node.getId()))
+            .andExpect(status().isInternalServerError())
```

Comment:
I don't get why this should be a internal Error...

Reply:
It's because the mockResponse code of modification-server call here is 404, and then it throws a StudyException with type GET_MODIFICATIONS_FAILED, which is mapped into INTERNAL_SERVER_ERROR status in RestResponseEntityExceptionHandler.java

Reply:
I agree it's a bit weird here, it's more of a forbidden request than a internal server error, maybe change the Exception Handler ?